examples/ios/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+use askar_crypto::{alg::p256::P256KeyPair, repr::KeyPublicBytes};
```

Comment:
Is verification the only reason for including Askar? If so, is it worth adding a dependency for something we can also handle outside this lib?

Reply:
This is for the example to show that verification works. Askar is just used here for ease of use. Its more of a dev dependency and won't be included in the lib.

Reply:
Perfect!